actions/ci-lint-ts/action.yml

```diff
@@ -76,6 +76,13 @@ runs:
       shell: bash
       run: pnpm run lint:affected --base=${{ inputs.base-ref }}
 
+    - name: Store lint result artifacts
+      if: always()
+      uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+      with:
+        name: ts-lint-results
+        path: ./eslint-report.json
```

Comment:
I don't see any of the repos using this action creating any format of report, so I'm assuming eslint as the standard and going with it

Reply:
hmm maybe instead of running `pnpm lint` and storing the results (which I don't think it's doing?) we can use this action instead?

https://github.com/marketplace/actions/lint-action

it seems like it has annotation as well - what do you guys think @chainchad @erikburt

Reply:
for eslint we will need to specify the output no?
https://eslint.org/docs/latest/use/command-line-interface

Reply:
yes, we expect pnpm `lint` command in the repo using the action to provide the output, I've added an input parameter `eslint-json-report-paths` that they can use as well

Reply:
Should they be allowed to specify which lint command to run, and then also the expected path to the report?

Reply:
I've come across some repos where the eslint report is generated but with a different artifact or file name, I've also added some defaults in case they don't want to use them (they wouldn't need to provide these inputs if they use our `ci-test-ts` and `ci-lint-ts`)

Reply:
I'm resolving this conversation. if requested changes, I will implement them in separate PR